dts/arm/nordic/nrf52840.dtsi

```diff
@@ -55,6 +55,33 @@
 			label = "UART_1";
 		};
 
+		gpiote0: gpiote@40006000 {
+			compatible = "nordic,nrf-gpiote";
+			reg = <0x40006000 0x1000>;
+			status = "disabled";
+			label = "GPIOTE_0";
```

Comment:
What could be interesting here is to add a channel and bind it to a gpio pin.

Reply:
I left this for a moment, with the idea of expanding the capabilities later as they are needed